app/code/community/Hps/Securesubmit/Model/Paypal.php

```diff
@@ -306,8 +306,13 @@ public function isAvailable($quote = null)
         if($quote && $quote->getBaseGrandTotal()<$this->_minOrderTotal) {
             return false;
         }
-
-        return $this->getConfigData('secretapikey', ($quote ? $quote->getStoreId() : null))
+        
+        $storeId = $quote ? $quote->getStoreId() : null;
+        return $this->getConfigData('username', $storeId)
+            && $this->getConfigData('password', $storeId)
+            && $this->getConfigData('device_id', $storeId)
+            && $this->getConfigData('license_id', $storeId)
+            && $this->getConfigData('site_id', $storeId)
```

Comment:
@SenthilKumar-86 is it really safe to return a bool here instead of the API Key?

Reply:
@ttnnkkrr The method returns a boolean casted by the use of `&&`.

Reply:
@slogsdon Yes but it looked like it used to return API key I was questioning if that was intended

Reply:
@ttnnkkrr The method has always returned a boolean thanks to the inclusion of `&& parent::isAvailable($quote)` just after this. That `&&` casts the value returned from `$this->getConfigData` to a boolean now, just as it did before.

Reply:
@slogsdon  I see now i missed the end of that old statement being && parent::isAvailable($quote);